### ISSUE TITLE
 tf.image.resize_bilinear ops have not support

### ISSUE THREAD

**Issue by hufangjian** [tensorflow/tensorflow#24161]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installed from (source or binary): binary
- TensorFlow version (or github SHA if from source):1.12

i want to convert deeplabv3  mobilenet to tflite but it do not support?
when will you add the 'tf.image.resize_bilinear' ops in tensorflow lite?thank you.

**Comment by achowdhery** [comment]
We do have ResizeBilinear op in TensorFlow Lite.
Also, please see below if the op you are interested is not same as the version in TFLite: 

As we work toward fleshing out the builtin op library for TensorFlow Lite, we've been working on an experimental feature that allows using select TensorFlow ops from within the TensorFlow Lite runtime. The goal is to help reduce some of the friction for using models that rely on ops not yet natively supported by TensorFlow Lite (at the cost of increased binary size). This feature requires opting in during model conversion, as well as adding an additional dependency. More details can be found [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/using_select_tf_ops.md).

Feedback is very much appreciated (either via GitHub or directly via tflite@tensorflow.org), and we'll be adding and refining functionality over the coming weeks. Cheers.

**Comment by jdduke** [comment]
Actually, `ResizeBilinear` should already be supported. Can you paste the output from your conversion attempt?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by jdduke** [comment]
The nightly TF/TFLite builds should have resize_bilinear, and it will also be in the upcoming 1.13 release.

**Comment by vatsDivyank** [comment]
System Information:
TensorFlow 2.3
Model Architecture - MobileNet V1
Arduino_TensorFlow Lite 2.4.0 Alpha
I fined Tuned Mobilenet V1 for face recoginition and later quantized using full integer Quantization Technique and later converted it to .cc format to upload it to Arduino Nano 33 BLE Sense. I used all_ops_resolver function but when i ran the Model on Microcontroller, invoke failed() and error message which was shown on Serial Monitor was " **Didn't find op for builtin opcode 'RESIZE_BILINEAR' version '3'. An older version of this builtin might be supported. Are you using an old TFLite binary with a newer model?**". Can anyone suggest me, how i can work around this error message.
